applications/FemToDemApplication/python_scripts/CouplingFemDem3D.py

```diff
@@ -136,6 +143,8 @@ def SolveSolutionStep(self):
 
 #============================================================================================================================
 	def GenerateDEM(self): # 3D version
+		if self.echo_level > 0:
+			KratosMultiphysics.Logger.PrintInfo("FEM-DEM:: GenerateDEM")

```

Comment:
I don't know of there is a conditional KratosInfo on python, I would approve, but check it please

Reply:
I've serached in Kratos and at least no one is using it.. :S

Reply:
I've searched in Kratos and no one is using it at least... :S

Reply:
I don't think this exists in python